report/clients/bundle.js

```diff
@@ -15,3 +15,4 @@ export {DOM} from '../renderer/dom.js';
 export {ReportRenderer} from '../renderer/report-renderer.js';
 export {ReportUIFeatures} from '../renderer/report-ui-features.js';
 export {renderReport} from '../renderer/api.js';
+export {swapLocale, format} from '../../shared/localization/i18n-module';
```

Comment:
what uses the esm report? I forget, but it uses `bundle.js` file too so it will have these in there now

Reply:
DevTools uses bundle.esm.js.  
 (old psi used a psi.js build that was also esm, but that's dead now. new psi uses bundle.umd.js)

Reply:
so this adds code to the devtool report bundle, when it isn't needed there. should we avoid that?

or should we add swap locale to LH panel :)

Reply:
addressed in [68935ef](https://github.com/GoogleChrome/lighthouse/pull/13885/commits/68935ef1a8f03de861b3c5da277d09db11ea21f6)